Source/ZoomNet/Models/ZoomRoomIssueDetails.cs

```diff
@@ -34,14 +34,14 @@ public class ZoomRoomIssueDetails
 		/// * Scheduling Display.<br/>
 		/// </summary>
 		/// <value>The Issue name.</value>
-		[JsonProperty(PropertyName = "issue_name")]
-		public IssueType IssueType { get; set; }
+		[JsonProperty(PropertyName = "issue")]
+		public IssueType Issue { get; set; }
```

Comment:
I realize that the json element from Zoom is called "issue" but I think "IssueType" or simply "Type" or something along those line is a more suitable name for this property in our C# model to describe what this property contains.